packages/r/retroshare/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+releases:
+  id: ~ # No id on https://release-monitoring.org as of 2024-11-01
```

Comment:
The project has been added [here](https://release-monitoring.org/project/375304/) :tada: